src/main/java/frc/robot/subsystems/superstructure/arm/ArmIOReal.java

```diff
@@ -126,8 +126,6 @@ public void config() {
         rightTalonFXConfiguration.TorqueCurrent.PeakReverseTorqueCurrent = -80;
         rightTalonFXConfiguration.CurrentLimits.StatorCurrentLimit = 60;
         rightTalonFXConfiguration.CurrentLimits.StatorCurrentLimitEnable = true;
-        rightTalonFXConfiguration.Feedback.FeedbackSensorSource = FeedbackSensorSourceValue.FusedCANcoder;
-        rightTalonFXConfiguration.Feedback.FeedbackRemoteSensorID = pivotCANCoder.getDeviceID();
         rightTalonFXConfiguration.Feedback.RotorToSensorRatio = armConstants.pivotGearing();
```

Comment:
`RotorToSensorRatio` isn't needed if not using `FusedCANcoder`. Also, this probably needs a `SensorToMechanismRatio`.